### DISCUSSION TITLE
multiple trackers

### DISCUSSION THREAD

**Discussion by jesperordrup** [umami-software/umami#2573] [General]
### Describe the feature or enhancement

Is it possible to add two website trackers to one homepage?

usecase

solution with many sites
abc.dk/site1
abc.dk/site2
abc.dk/sitex

I need to track indivually and an accumulated / roll up. Its a typical scenario for - us at least.

Related to that: when creating a website tracker, you need to provide a domain, not an url. Is that needed logically?

I cant use it to accuratly describe the target abc.dk/site1 as it invalidates anything not domain.

But I dont even need it - unless some logic really needs it`?

**Comment by mikecao** [comment]
The domain name is used to filter out referrers. All that really matters is the website id. So for your situation, you can add 3 websites and render a different tracker for each url.
